addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Hi @kaikreuzer and @daveoxley

Thanks for resolving some of these issues so promptly. 

Simon Overton @soverton100 has show some interest in picking this up over in the openHAB forums https://community.openhab.org/t/new-clipsal-c-bus-binding/9208/8?u=neohusky

Of course @daveoxley is more than welcome to start exploring/playing with openHAB and maybe make some further contribution in addition to his awesome library ;)